.github/workflows/build.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build
+on: [push]
+
+jobs:
+  build:
+    name: Build SemTK
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - shell: bash
+        name: Build SemTK
+        run: |
+          . .env
```

Comment:
The SemTK poms don't use environment variables, so there's no need to source .env here.

Reply:
Okay, thanks for the pointer! I'll try removing this.